### PR TITLE
feat: add a template for issues

### PR DESCRIPTION
**Pull request by thephez**
This will provide an general issue template to prompt for information when someone wants to open a new issue.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (shumkov) [inline .github/ISSUE_TEMPLATE/issue.md:1]: Can we do labels which are depending on the issue type, like "bug" or "feature"?

Comment (thephez) [reply .github/ISSUE_TEMPLATE/issue.md:1]: It is possible to define multiple issue templates for different purposes which require users to select an issue type (like [this](https://github.com/thephez/exp/issues/new/choose)). I was trying to keep it simple/generic. We could have 2 options though and still keep it generic enough to use on most repos.

Comment (thephez) [reply .github/ISSUE_TEMPLATE/issue.md:1]: @shumkov Opened alternative PR #47 with 2 templates that include labels.

Comment (shumkov) [reply .github/ISSUE_TEMPLATE/issue.md:1]: Awesome! We need more guys to check it before we merge it.

**Comment by thephez** [comment]
Closing this one in favor of #47

**Pull request closed without merging**